app/routers/classes.py

```diff
@@ -0,0 +1,58 @@
+from fastapi import APIRouter
+
+from ..database import mappings_col
+from ..schema import AbstractLabelMapping
+from ..classifier import get_classes
+
+router = APIRouter(
+    tags=["classes"],
+    responses={404: {"description": "Not Found"}},
+)
+
+
+def store_classes():
+    """
+    Stores all the classes from `get_classes()` into the database at startup.
+
+    """
+    classes = get_classes()
+    for model_class in classes:
+        query = {"modelClass": model_class}
+        existing_class = mappings_col.find_one(query)
```

Comment:
Bad idea, you have nested loops here, what you should do instead here is:
1. Create a unique index on the internal names
2. Try inserting a new entry 
3. Catch duplicate key errors and ignore them, also consider moving this function to `crud.py`

Reply:
got it thanks alot.